worker/include/manager_service.h

```diff
@@ -14,8 +14,13 @@ namespace ava_manager {
 
 class ManagerServiceServerBase {
 public:
+  /**
+   * @brief Start a manager service that will exec a worker when a connection is accepted
+   * @param worker_argv arguments to forward to exec
+   * @param worker_argc number of arguments in worker_argv
+   */
   ManagerServiceServerBase(uint32_t manager_port, uint32_t worker_port_base,
-      std::string worker_path);
```

Comment:
I think it would be nice to keep `worker_path` as an essential argument and not include it in `worker_argv`. It's not needed to be sent to API server as well.

Reply:
This is the same reason why I'm stilling thinking about #74 and haven't merged it.

Reply:
oops. these are essentially the same. I forgot I opened that PR

Reply:
worker path is still included in `worker_argv` and must always be the first entry. I think I refactored all uses of it.

Reply:
Since I changed the type of exec call we don't have to check if worker_path is valid any more, or pass it separately to exec, so keeping it around is less important now.

Reply:
Seems not really matter as we will finally use some arg_parser and specify worker's argv using json file or string.